util/ctags.sh

```diff
@@ -11,9 +11,9 @@
 # Usage: ./util/ctags.sh [...arguments for ctags...]
 #
 # This script runs ctags twice. In the first pass, ctags extract macro
```

Comment:
If we're fixing grammar here, it should be `ctags extracts macro...`, since `ctags` is the (singular) name of the tool, despite ending with a `s` (like the correct `ctags reads...` on line 15)